src/thunderbots/software/network_input/backend.cpp

```diff
@@ -172,7 +182,17 @@ Team Backend::getFilteredFriendlyTeamData(std::vector<SSL_DetectionFrame> detect
             robot_detection.confidence = friendly_robot_detection.confidence();
             robot_detection.timestamp  = Timestamp::fromSeconds(detection.t_capture());
 
-            friendly_robot_detections.push_back(robot_detection);
+
+            bool ignore_robot =
+                Util::DynamicParameters::AI::refbox::ignore_camera_data.value() &&
+                (Util::DynamicParameters::AI::refbox::min_valid_x.value() >
+                     robot_detection.position.x() ||
+                 Util::DynamicParameters::AI::refbox::max_valid_x.value() <
+                     robot_detection.position.x());
```

Comment:
Similar comments to above about:
- breaking this out into two booleans to be more readable
- change `ignore_camera_data` to `ignore_invalid_camera_data`
- shouldn't we always ignore invalid data?

Reply:
Done. See above for always ignoring data.